src/App.vue

```diff
@@ -295,7 +297,9 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   @Getter(useStore) getUserLastName!: string
   @Getter(useStore) getUserEmail!: string
   @Getter(useStore) getUserPhone!: string
-  @Getter(useStore) isAmalgamationFiling!: boolean
+  // @Getter(useStore) isAmalgamationFilingHorizontal!: boolean
+  // @Getter(useStore) isAmalgamationFilingRegular!: boolean
+  // @Getter(useStore) isAmalgamationFilingVertical!: boolean
```

Comment:
Already imported via a mixin, but this shows a future developer that it would otherwise need to be declared here (instead of omitting this altogether and a developer won't know where the declaration came from).